e2e/hero-header.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test, expect } from '@playwright/test';
```

Comment:
We recently updated the Playwright Test library in the project, we should update all the locators (`page.locator()`) in these tests to the format inspired by testing library https://playwright.dev/docs/release-notes#locators in order to have uniform syntax across the project. The other tests are already using these new locators.

Reply:
Fixed via [85caacd](https://github.com/reactdeveloperske/reactdevske-website/pull/105/commits/85caacd910e2c9b76ce2389a6b91d81012814c36)